app/domain/authentication/authn_jwt/signing_key/fetch_public_keys_signing_key.rb

```diff
@@ -0,0 +1,24 @@
+
+module Authentication
+  module AuthnJwt
+    module SigningKey
+      # This class is responsible for parsing JWK set from public-keys configuration value
+      class FetchPublicKeysSigningKey
+
+        def initialize(
+          public_keys:,
+          logger: Rails.logger
+        )
+          @logger = logger
+          @public_keys = public_keys
+        end
+
+        def call(force_fetch:)
+          signing_keys = PublicSigningKeys.new(JSON.parse(@public_keys))
+          signing_keys.validate!
+          { keys: JSON::JWK::Set.new(signing_keys.value) }
```

Comment:
Can this two lines be in one line?

Reply:
I'm missing which two lines this comment is related...